kernel/packages/unity-interface/BrowserInterface.ts

```diff
@@ -466,6 +466,10 @@ export class BrowserInterface {
     }
   }
 
+  public async KillPortableExperience(data: { portableExperienceId: string }): Promise<void> {
```

Comment:
Is `kill` the only action here because the UI can't start new PEs yet?

Reply:
Yes!